TaffoInitializer/IndirectCallPatcher.h

```diff
@@ -0,0 +1,20 @@
+#ifndef TAFFO_INDIRECTCALLPATCHER_H
+#define TAFFO_INDIRECTCALLPATCHER_H
+
+#include <list>
+#include "llvm/IR/CallSite.h"
+#include "llvm/IR/Dominators.h"
+
+namespace taffo {
+
+/** Function type for handlers **/
+using handler_function = void (*)(const llvm::Module &m,
+                                  std::vector<llvm::Instruction *> &toDelete,
+                                  llvm::CallInst *curCallInstruction,
+                                  const llvm::CallSite *curCall,
+                                  llvm::Function *indirectFunction);
+
+void manageIndirectCalls(llvm::Module &m);
```

Comment:
Copy/Move documentation from cpp to header

Reply:
Added documentation to the header